src/isCircularChecker.java

```diff
@@ -0,0 +1,15 @@
+public class isCircularChecker {
+
+    public boolean isCircular(String s1,String s2) throws Exception{
```

Comment:
Better change Exception -> InvalidLengthException. A method should throw only expected exceptions